lib/ash_phoenix/form_data/error.ex

```diff
@@ -14,7 +14,16 @@ end
 
 defimpl AshPhoenix.FormData.Error, for: Ash.Error.Query.InvalidQuery do
   def to_form_error(error) do
-    {error.field, error.message, error.vars}
+    error_fields =
+      if is_nil(error.field) do
+        Map.get(error, :fields, [])
+      else
+        [error.field]
+      end
+
+    for field <- error_fields || [] do
+      {field, error.message, error.vars}
+    end
```

Comment:
```suggestion
    fields = List.wrap(error.field || Map.get(error, :fields) || [])

    for field <- fields do
      {field, error.message, error.vars}
    end
```

My suggestion to shorten the code.

Reply:
Looks better with your suggestion 👍